### ISSUE TITLE
Sidebar design improvements

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/richdocuments#2048]
- Design with Nextcloud sidebar open
- Possibly integrate sidebar better (e.g. no preview)

**Comment by juliushaertl** [comment]
@jancborchardt I also had some discussion today about this with Collabora before you joined our call about possibilities to merge the two sidebars. I think it would be interesting to think about that from a design perspective as well.

**Comment by juliushaertl** [comment]
Maybe we could have a shorter brainstorming call about this design wise, to more align about the actual efforts we want to take in that area.

**Comment by jancborchardt** [comment]
Yep, some calls ago (I think when you were on vacation) @pedropintosilva and I briefly talked about that.

We could use the style of the Collabora sidebar, well integrated. The content of the Nextcloud sidebar would be available as tabs.
The only issue is in e.g. Presentations, where the right sidebar already has tabs, we might have a space issue.

**Comment by juliushaertl** [comment]
Yes, also this might be a quite large task tue to technical challenges. Either we need to have a way that collabora can expose its sidebar to Nextcloud or the other way round, where Nextcloud would need a sidebar only page that could then be rendered within an iframe in Collabora, but that may also be challenging, especially when it comes to actions in the sidebar that shall be executed.

Any chance that we could maybe start off here with some mockups to see in which direction we could go for the UI?
